cvat/apps/lambda_manager/serializers.py

```diff
@@ -5,30 +5,42 @@
 from drf_spectacular.utils import extend_schema_serializer
 from rest_framework import serializers
 
+class AttributeMappingSerializer(serializers.JSONField):
+    def validate(self, attrs):
+        for attr_name in self.initial_data:
+            if not isinstance(self.initial_data[attr_name], str):
+                raise serializers.ValidationError(f"Not valid field value type: {attr_name}, must be string")
+        return attrs
+
+class SkeletonLabelMappingEntrySerializer(serializers.DictField):
```

Comment:
Shouldn't the parent class be `Serializer`?

Reply:
Implemented the same way as it was done for LabelMappingEntrySerializer

Reply:
That doesn't answer the question, though. For all I know, `LabelMappingEntrySerializer` is wrong too.

Reply:
I don't know if it should not or not, I just did like it was